src/lib.rs

```diff
@@ -68,15 +68,20 @@ pub enum DisplayState {
 ///[`set_config`]: struct.HexView.html#method.set_config
 #[derive(Debug, Clone, Copy)]
 pub struct HexViewConfig {
-    /// Controls the number of bytes per line. It needs to be greater than 0 and equal or higher than the bytes_per group value. Default is 16
+    /// Controls the number of bytes per line. It needs to be greater than 0 and equal or higher than the `bytes_per_group` value.
+    /// Default is 16.
```

Comment:
Unfortunately this isn't still quiet right. Please look at other doc example in the code and/or use the following as an example of what I mean:

```
    /// Controls the number of bytes per line.
    ///
    /// It needs to be greater than 0 and equal or higher than the `bytes_per_group` value.
    /// Default is 16.
    pub bytes_per_line: usize, 
```

As said, if you're unsure, use `cargo doc --open` and see if it looks like the other examples.